journalpump/journalpump.py

```diff
@@ -675,6 +724,18 @@ def _truncate_long_message(self, json_entry):
         }
         return json.dumps(entry, default=default_json_serialization).encode("utf8")
 
+    def _apply_secret_filters(self, data):
+        if self.reader.secret_filter_metrics is True:
```

Comment:
Nit: PEP-8 says this should be `if self.reader.secret_filter_metrics` but more importantly this entire check should be removed (or perhaps rather replaced with `if self.reader.secret_filters`), now if config says metrics for secret filtering shouldn't be reporting the code ends up skipping the filtering altogether instead of just the reporting part.